exporter/opentelemetry-exporter-jaeger/tests/test_jaeger_exporter_protobuf.py

```diff
@@ -66,20 +69,23 @@ def test_constructor_by_environment_variables(self):
                 OTEL_EXPORTER_JAEGER_ENDPOINT: collector_endpoint,
                 OTEL_EXPORTER_JAEGER_CERTIFICATE: os.path.dirname(__file__)
                 + "/certs/cred.cert",
+                # OTEL_RESOURCE_ATTRIBUTES: "service.name=my-opentelemetry-jaeger",
```

Comment:
Why comment this out? Aren't we testing the constructor service name being populated by env var?

Reply:
patching of env var is being done in distro tests, hence did not cover it here.If you want one of the tests in jaeger to cover it, I will modify this testcase.

Reply:
I'd prefer to have explicit tests per each test file to cover each use case.